<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VAIDEHI-PC" ignored="3" name="ValidateSignUPTest" tests="1" failures="1" timestamp="03 Jan 2019 06:00:32 GMT" time="20.504" errors="0">
  <testcase name="signUp" time="2.624" classname="day6.FifthCS">
    <failure type="java.lang.Exception" message="UserID already exists">
      <![CDATA[java.lang.Exception: UserID already exists
at day6.FifthCS.signUp(FifthCS.java:77)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- signUp -->
  <testcase name="verifyLoginToEdureka" time="0.0" classname="day6.FifthCS">
    <ignored/>
  </testcase> <!-- verifyLoginToEdureka -->
  <testcase name="browseForAllCourses" time="0.0" classname="day6.FifthCS">
    <ignored/>
  </testcase> <!-- browseForAllCourses -->
  <testcase name="logout" time="0.0" classname="day6.FifthCS">
    <ignored/>
  </testcase> <!-- logout -->
</testsuite> <!-- ValidateSignUPTest -->
